specification/marketplacecatalog/data-plane/Reviews/preview/2023-01-01-preview/reviews.json

```diff
@@ -0,0 +1,405 @@
+          {
+            "type": "string",
+            "name": "productId",
+            "in": "path",
+            "required": true,
+            "description": "Unique Product Id",
+            "x-nullable": true
+          },
+          {
+            "type": "string",
+            "name": "skipToken",
+            "in": "query",
+            "description": "Skip token to continue getting comments from a given point",
+            "x-nullable": true
+          },
+          {
+            "type": "integer",
+            "name": "pageSize",
+            "in": "query",
+            "description": "Maximum number of reviews to return",
+            "format": "int32",
+            "default": 50,
+            "x-nullable": true
+          },
+          {
+            "type": "string",
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "default": "2023-01-01-preview",
+            "x-nullable": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "x-nullable": false,
+            "description": "Returns a list of reviews for the given unique product",
+            "schema": {
+              "$ref": "#/definitions/PageResultOfReviewDto"
+            }
+          },
+          "400": {
+            "x-nullable": false,
+            "description": "In case request is malformed",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "500": {
+            "x-nullable": false,
+            "description": "In case an error encountered",
+            "schema": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "/products/{productId}/reviews/{reviewId}/comments": {
+      "get": {
+        "tags": [
+          "Reviews Public API"
+        ],
+        "summary": "Get the comments of a given review",
+        "operationId": "ReviewComment_GetComments",
+        "x-ms-examples": {
+          "Facets_Get": {
+            "$ref": "./examples/GetReviewComments.json"
+          }
+        },
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "type": "string",
+            "name": "reviewId",
+            "in": "path",
+            "required": true,
+            "description": "Review Id",
+            "x-nullable": true
+          },
+          {
+            "type": "string",
+            "name": "productId",
+            "in": "path",
+            "required": true,
+            "description": "Unique Product Id",
+            "x-nullable": true
+          },
+          {
+            "type": "string",
+            "name": "skipToken",
+            "in": "query",
+            "description": "Skip token to continue getting comments from a given point",
+            "x-nullable": true
+          },
+          {
+            "type": "integer",
+            "name": "pageSize",
```

Comment:
You should conform to the parameter names in the guidelines, so this should become `maxpagesize`
See https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#collections

Reply:
Done

Reply:
Thanks for addressing @nrafalowski. For backward compat & consistency reasons the casing must be `maxpagesize` all lowercase even though it violates the lowerCamelCase rule, this parameter casing is baked into the autorest tooling afaik.

Reply:
done